apps/hash-graph/lib/graph/src/ontology/entity_type.rs

```diff
@@ -259,10 +259,10 @@ pub enum EntityTypeQueryPath {
     /// Only used internally and not available for deserialization.
     VersionId,
     /// Only used internally and not available for deserialization.
-    Schema,
+    Schema(Option<JsonPath<'p>>),
```

Comment:
What would be an easy way to test this locally now that this is not deserializable?

Reply:
You probably just have to remove `#[sersde(skip)]` on the `EntityTypeQueryToken::Schema`

Reply:
Thanks for spotting this! I fixed that in https://github.com/hashintel/hash/pull/1956/commits/3e981db0627da564b5eead724e816b3e6f2d198b. The approach is not necessarily pretty, but it's out of scope to solve this problem in this PR.

Reply:
Awesome, thanks! Works now :)